docs/synchronizations-faq.md

```diff
@@ -0,0 +1,43 @@
+# Synchronizations FAQ
+##
+
+In Private Packagist synchronizations is how you keep packages, teams, their members, and access permissions in sync with your version control system like GitHub, GitLab or Bitbucket.
+
+You can find below answers for frequently-asked questions concerning synchronizations.
+
+## What happens if you promote a synchronization to a primary synchronization?
+
+Private Packagist organizations that are synchronized with a remote organization have automatically all admins and owners of the synchronization assigned to the admins and owners team in Private Packagist.
+
+If an organization is synchronized with multiple remote organizations then only admins and owners of the primary synchronization are matched to the respective teams.
+
+When promoting a synchronization to a primary synchronization, new admins and owners will be added from the primary synchronization to Private Packagist.
+In case you change the integration for the new primary synchronization (e.g. from GitHub to Bitbucket), admins and owners who are not connected to the new service yet will lose admin/owner access to the organization. To restore their access, the user have to connect their account to the new service from the profile page.
```

Comment:
```suggestion
In case you change the integration for the new primary synchronization (e.g. from GitHub to Bitbucket), admins and owners who are not connected to the new service yet will lose admin/owner access to the organization. To restore their access, the users have to connect their account to the new service from the profile page.
```